packages/documentation/src/stories/components/popover/popover.stories.ts

```diff
@@ -46,6 +47,14 @@ const meta: MetaComponent = {
         category: 'General',
       },
     },
+    maxWidth: {
+      name: 'Max width of the popover',
+      description:
+        'Value can either be in `vw`, `px` or `%`. If no max-width is defined, the popover will extend to the width of its content.',
+      table: {
+        category: 'General',
```

Comment:
Could be helpful to know for some people.

```suggestion
        category: 'General',
        defaultValue: { summary: '280px' }
```